src/Altinn.Notifications/Controllers/EmailNotificationOrdersController.cs

```diff
@@ -60,7 +61,7 @@ public async Task<ActionResult<OrderIdExt>> Post(EmailNotificationOrderRequestEx
             return ValidationProblem(ModelState);
         }
 
-        string? creator = User.GetOrg();
+        string? creator = HttpContext.Items["Org"] as string;
```

Comment:
Is this defined in both cases? I wish we had something less magic, but we can save that for later.